modules/osm/way.js

```diff
@@ -138,31 +138,53 @@ Object.assign(osmWay.prototype, {
     },
 
 
-    isOneWay: function() {
-        // explicit oneway tag..
-        var values = {
-            'yes': true,
-            '1': true,
-            '-1': true,
-            'reversible': true,
-            'alternating': true,
-            'no': false,
-            '0': false
-        };
-        if (values[this.tags.oneway] !== undefined) {
-            return values[this.tags.oneway];
+    /** @returns {boolean} for example, if `oneway=yes` */
+    isOneWayForwards() {
```

Comment:
As these 3 function are almost identical: would be worth refactoring out the common parts into a helper fucntion and call that one with the data respectively?

 ```js
function checkOneWayTags(entityTags, tagList) {
    …
}

…

    isOneWayForwards() {
        return checkOneWayTags(this.tags, osmOneWayForwardTags);
    }.
```

Reply:
done, I've made it even more generic than that, because we use this structure all over the codebase (`osmRightSideIsInsideTags`, `osmPavedTags`, ...)